src/scripts/add_models_to_leaderboard.py

```diff
@@ -0,0 +1,91 @@
+"""Script which searches the huggingface_hub for models fitting the supported tasks and add their results to the leaderboard."""
+
+import pandas as pd
+from huggingface_hub.hf_api import HfApi, ModelFilter
+
+from alexandra_ai_eval.evaluator import Evaluator
+from alexandra_ai_eval.task_configs import get_all_task_configs
+
+
+def main():
+    evaluator = Evaluator()
+    task_mapping = get_all_task_configs()
+
+    searches = []
+    languages = ["da", "multilingual", "no", "sv", "nn", "nb"]
+    for task_name, task_config in task_mapping.items():
+        search = {task_name: []}
+        for language in languages:
+            # Get supertask, and check that it is correct.
+            if task_config.supertask == "sequence-classification":
+                supertask = "text-classification"
+            else:
+                supertask = task_config.supertask
+
+            # Add search terms, some tasks have no search terms, as they are examples of general supertasks.
+            if not task_config.search_terms:
+                search[task_name].append(
+                    {
+                        "search": "",
+                        "filter": ModelFilter(task=supertask, language=language),
+                    }
+                )
+            else:
+                for search_term in task_config.search_terms:
+                    search[task_name].append(
+                        {
+                            "search": search_term,
+                            "filter": ModelFilter(task=supertask, language=language),
+                        }
+                    )
+
+        searches.append(search)
+
+    # Loop through searches
+    hf_api = HfApi()
+    models_ids_evaluated = []
```

Comment:
Should this be initialised using the cache?

Reply:
I am not sure what you mean?

Reply:
As in, if you run this script and it fails halfway through, then it will have stored the evaluations in the cache file. Next time we run this script it could load in the cache file and populate `models_ids_evaluated` with the models already evaluated?